packages/sdk/typescript/human-protocol-sdk/test/escrow.test.ts

```
@@ -0,0 +1,1368 @@
+      const escrowAddress = ethers.constants.AddressZero;
+      const recipients = [FAKE_ADDRESS];
+      const amounts = [BigNumber.from(100)];
+      const finalResultsUrl = VALID_URL;
+      const finalResultsHash = FAKE_HASH;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+
+      await expect(
+        escrowClient.bulkPayOut(
+          escrowAddress,
+          recipients,
+          amounts,
+          finalResultsUrl,
+          finalResultsHash
+        )
+      ).rejects.toThrow(new InvalidEthereumAddressError(FAKE_ADDRESS));
+    });
+
+    test('should throw an error if url is empty string', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+      const recipients = [ethers.constants.AddressZero];
+      const amounts = [BigNumber.from(100)];
+      const finalResultsUrl = '';
+      const finalResultsHash = FAKE_HASH;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+
+      await expect(
+        escrowClient.bulkPayOut(
+          escrowAddress,
+          recipients,
+          amounts,
+          finalResultsUrl,
+          finalResultsHash
+        )
+      ).rejects.toThrow(ErrorUrlIsEmptyString);
+    });
+
+    test('should throw an error if final results url is invalid url', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+      const recipients = [ethers.constants.AddressZero];
+      const amounts = [BigNumber.from(100)];
+      const finalResultsUrl = FAKE_URL;
+      const finalResultsHash = FAKE_HASH;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+
+      await expect(
+        escrowClient.bulkPayOut(
+          escrowAddress,
+          recipients,
+          amounts,
+          finalResultsUrl,
+          finalResultsHash
+        )
+      ).rejects.toThrow(ErrorInvalidUrl);
+    });
+
+    test('should throw an error if hash is empty string', async () => {
```

fix writing: 
'should throw an error if hash is an empty string',